### ISSUE TITLE
Cropped Image Editor: Support Polygon Boundaries

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#16]
## Feature description

Follow-up issue for ampproject/amp-wp#3964 

Currently, the image cropping when rotating does not have boundaries due to Moveable not supporting polygon boundaries.

We will need to investigate if it's feasible to implement this via contributing directly to [Moveable ](https://github.com/daybrush/moveable)to support that, and based on that either
a) Implement the support, or
b) Not allow cropping while rotated.

See also conversation starting from [here](https://github.com/ampproject/amp-wp/pull/3964#discussion_r360246351).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Understand if it's possible to contribute to [Moveable](https://github.com/daybrush/moveable) to support polygon boundaries 
* Based on the findings, either contribute, or remove support for cropping while rotated
* Implement the boundaries for the Stories.

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

**Comment by dvoytenko** [comment]
Filed https://github.com/daybrush/moveable/issues/163

**Comment by swissspidy** [comment]
@miina It seems like this was since added in Moveable? See https://github.com/daybrush/moveable/issues/163#issuecomment-594765982

**Comment by miina** [comment]
Looks like it, I guess the issue is unblocked now for fixing it in the editor.

**Comment by swissspidy** [comment]
@miina @merapi Is this still relevant? If so, might be worth revisiting. Otherwise we should probably just close this since this hasn't been updated in 2 years.

**Comment by miina** [comment]
Not relevant anymore, closing.
